about.html

```diff
@@ -0,0 +1,109 @@
+        </div>
+        <div class="logo">
+            <a href="./index.html "> <img src="./images/logo_bogota.jpg" alt="logo" /> </a>
+        </div>
+        <ul class="menu-deployed">
+            <li><a href="./index.html " onClick="menuClose(1);">Home</a></li>
+            <li><a href="#program " onClick="menuClose(1);">Program</a></li>
+            <li><a href="#speakers " onClick="menuClose(1);">Heros</a></li>
+            <li><a href="#sponsors" onClick="menuClose(1);">Sponsor</a></li>
+
+
+            <!--   <li class="mail">
+                <img src="images2/union.png" alt="email" />
+            </li> -->
+        </ul>
+
+    </header>
+
+    <div id="mobile_menu">
+        <div id="mobile_button" class="close_button" onClick="menuClose(1);">
+            <img src="images/mobil_close_menu.png" alt="close button" />
+        </div>
+        <ul>
+            <li><a href="./index.html " onClick="menuClose(1);">Home</a></li>
+            <li><a href="#program " onClick="menuClose(1);">Program</a></li>
+            <li><a href="#speakers " onClick="menuClose(1);">Heros</a></li>
+            <li><a href="#sponsors" onClick="menuClose(1);">Sponsor</a></li>
+        </ul>
+    </div>
+    <main>
+
+        <section id="cover" class="cover">
+         
+            <h2><img src="./images/main_big.png" alt="title" /> </h2>
+            
+
+            <div class="cover_extra">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc dapibus diam sed pulvinar congue. Vivamus
+                sit amet nisi a tellus interdum placerat vitae sit amet dolor. Praesent placerat hendrerit mi, in
+                ultrices lectus vulputate et. Aenean lacinia feugiat ultrices. Nullam dapibus tortor at purus eleifend
+                bibendum. Etiam tristique, quam eleifend auctor bibendum, nunc massa condimentum odio, et facilisis elit
+                odio in felis. Vivamus tincidunt est tempus orci tempus, non cursus ex maximus. Pellentesque habitant
+                morbi tristique senectus et netus et malesuada fames ac turpis egestas.
+
+                Fusce at dui a mauris sollicitudin accumsan. Aenean gravida eget metus ut porta. Ut at varius eros.
+                Aenean ac lectus molestie, varius ante at, faucibus dolor. Sed convallis lectus turpis, ac egestas
+                libero accumsan ac. Vivamus vulputate massa nec sapien cursus facilisis. Quisque turpis diam, porttitor
+                sed metus a, sodales placerat sem. Duis nec ipsum et ipsum lobortis lobortis. Fusce euismod blandit
+                neque non accumsan. Sed vulputate venenatis nibh, et sagittis sem pretium id. Nulla at dapibus velit.
+                Proin nec nisi porttitor, egestas sapien a, elementum.
+            </div>
+
+
+            <p>2022.02.18(SAT)- 20(MON) </p>
+            <p>Gran Salon Corferias, Bogotá-Colombia</p>
+
+
+
+        </section>
+        <section class="news">
```

Comment:
- [ ] The pages should look almost identical to the original design, outside the content differences, please consider adjusting the margins/paddings so your page is identical to the CC design